CONTRIBUTING.md

```diff
@@ -1,55 +1,54 @@
-# Contributing to @bflad Cookbooks
+# Contributing to Parallels Cookbooks
 
-Below is a modified version of Opscode's CONTRIBUTING.md in their public cookbooks. Thanks to them for the great documentation!
+We're glad you want to contribute to Parallels Cookbooks! The first
+step is the desire to improve the project.
 
-## Quick-contribute
+## Quick Contributing Steps
 
-* Use the testing methods and tools outlined below
-* Use Github issues and pull requests when possible
+1. Create an account on [GitHub](https://github.com).
+2. Create a pull request for your change on [GitHub](https://github.com).
+3. The community cookbook maintainers will review your change, and either merge
+the change or offer suggestions.
 
-I regularly review contributions and will get back to you if I have
-any suggestions or concerns.
+## The Apache License and the CLA/CCLA
+Licensing is very important to open source projects. It helps ensure the
+software continues to be available under the terms that the author desired.
 
-## The Apache License
+Parallels uses [the Apache 2.0 license](https://www.apache.org/licenses/LICENSE-2.0)
```

Comment:
This is a very broad statement and should be avoided. Or if we really do so for all cookbooks, please state that explicitly.